src/components/icons/IconStar.tsx

```diff
@@ -22,9 +22,5 @@ import { IconMaterialWrapper, IconProps } from './IconWrapper';
 
 export function IconStar(props: Readonly<IconProps>) {
   // This is Material Symbols' "kid_star" icon
-  return (
```

Comment:
There are actually two versions of star

Reply:
now handled with the export of `isFilled`, same for `IconHome`